src/utils/FormElementMixin.js

```diff
@@ -94,6 +94,7 @@ export default {
             this.isFocused = false
             this.$emit('blur', $event)
             this.checkHtml5Validity()
+			if (this.localOnBlur) this.localOnBlur()									  
```

Comment:
Coz we can't add one more onBlur listener to template. So, it's not nice in this case to use mixins.